example/app/config.yml

```diff
@@ -79,7 +79,7 @@ default:
       username: cumulususer
     services:
       EcsTaskHelloWorld:
-        image: cumuluss/cumulus-ecs-task:1.2.3
+        image: cumuluss/cumulus-ecs-task
```

Comment:
should this be versioned still?

Reply:
If not, should `developing-workflow-tasks.md` be updated to match?

Reply:
And that reminds me, do we need to update the docs to show other users how to use this work around, or is this really a temporary fix.  If temporary fix, do we have a ticket to do the next steps? (allow multiple private repositories.)

Reply:
You're right, it should definitely still be versioned - I forgot it when adding it back in.

I think it would be a good idea to make some updates to documentation and make this a known addition.

As for future work... I don't know of anything planned right now. There are certainly more complete ways to write this functionality but I don't know if those align with our objectives. I'll ask @laurenfrederick  or @bilts and follow up with you on that.